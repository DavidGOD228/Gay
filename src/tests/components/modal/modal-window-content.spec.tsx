import 'react-native'
// Note: test renderer must be required after react-native.
import React from 'react'
import renderer from 'react-test-renderer'
import ModalWindowContent from '../../../components/modal/modal-window-content'
import {Text} from 'react-native'

it('renders correctly empty', () => {
  const tree = renderer.create(
    <ModalWindowContent/>
  )
  expect(tree)
    .toMatchSnapshot()
})

it('renders correctly text', () => {
  const tree = renderer.create(
    <ModalWindowContent>
      <Text>
        All you had to do was follow the damn train CJ
      </Text>
    </ModalWindowContent>
  )
  expect(tree)
    .toMatchSnapshot()
})
