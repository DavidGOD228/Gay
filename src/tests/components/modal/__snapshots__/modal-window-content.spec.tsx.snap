// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly empty 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "flex-start",
    }
  }
>
  <View
    style={
      Object {
        "paddingLeft": 40,
        "paddingRight": 40,
      }
    }
  />
</View>
`;

exports[`renders correctly text 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "flex-start",
    }
  }
>
  <View
    style={
      Object {
        "paddingLeft": 40,
        "paddingRight": 40,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
    >
      All you had to do was follow the damn train CJ
    </Text>
  </View>
</View>
`;
